docs/getting-started/licensing/license-and-legal.md

```diff
@@ -1,27 +1,44 @@
 ---
 category: licensing
 meta-title: Editor license | CKEditor 5 Documentation
-meta-description: Choosing the right licence type for you needs.
+meta-description: Choosing the right license type for you needs.
 menu-title: Editor license
 order: 30
 modified_at: 2024-11-20
 ---
 
 # Editor license and legal terms
 
-The following legal notices apply to CKEditor&nbsp;5 and all software from CKEditor&nbsp;5 Ecosystem included with it.
+CKEditor is dual-licensed. This means we make it available under a choice between an open source and a commercial license.
 
-Copyright (c) 2003–2024, CKSource Holding sp. z o.o. All rights reserved.
+## Open source license
 
-Licensed under the terms of [GNU General Public License Version 2 or later](http://www.gnu.org/licenses/old-licenses/gpl-2.0.html).
+The open source distribution of CKEditor is licensed under the terms of GPL 2+. If you plan to use CKEditor under this license, you can register your application using [the CKEditor GPL 2+ license registration form](https://info.ckeditor.com/l/1018062/2024-08-22/39jtg).
```

Comment:
A couple of times LanguageTool marked for me GPL license as `This phrase is redundant (‘L’ stands for ‘license’). Use simply “GPL”.` But I think it's quite common to use it this way.